latch_cli/docker_utils/__init__.py

```diff
@@ -0,0 +1,197 @@
+import functools
+from dataclasses import dataclass
+from enum import Enum, auto
+from pathlib import Path
+from textwrap import dedent
+from typing import List
+
+import yaml
+
+from latch_cli.types import LatchWorkflowConfig
+
+print = functools.partial(print, flush=True)
```

Comment:
again, why

Reply:
idk it was in another place -- I killed both.